src/scripts/main.js

```diff
@@ -1,3 +1,42 @@
 'use strict';
 
-// write code here
+const headings = document.querySelectorAll('li');
+
+const headingsTexts = [
+  'Fruit',
+  'Fish',
+  'Red',
+  'Yellow',
+  'Ocean fishes',
+  'Freshwater fish',
+];
+
+headings.forEach((item) => {
+  if (headingsTexts.includes(item.childNodes[0].textContent.trim())) {
```

Comment:
Accessing 'item.childNodes[0].textContent' assumes that the text node is the first child. This might not always be the case, especially if there are other elements or text nodes. Consider using 'item.textContent.trim()' directly if the list item contains only text, or ensure the correct node is selected.